sky/packages/sky/lib/src/gestures/events.dart

```diff
@@ -0,0 +1,114 @@
+import 'dart:ui' as ui;
+
+/// Dart-layer version of ui.Event
+class InputEvent {
+
+  InputEvent({ this.type }) : timeStamp = 0.0;
```

Comment:
I'm not sure I understand this one, can you elaborate? In general I was trying to disturb client code as little as possible.